submissions/nilamop/popup/css.css

```diff
@@ -0,0 +1,139 @@
+.popup:target {
+  visibility: visible;
+  opacity: 1;
+}
+
+.popup {
+  margin: 5px auto;
+  padding: 20px;
+  background: #fff;
+  /*border-radius: 5px;*/
+  border: 1px solid #d0d0d0;
+  box-shadow: 1px 12px 15px 1px #C0C0C0;
+  width: 297px;
+  position: relative;
+  left: 30%;
+  opacity: 0;
+  
+}
+
+.popup .close {
+  position: absolute;
+  top: -10px;
+  right: 0px;
+  transition: all 200ms;
+  font-size: 30px;
+  font-weight: bold;
+  text-decoration: none;
+  color: #333;
+}
+
+.popup .close:hover {
+  color: #06D85F;
+}
+
+.popup .content {
+  max-height: 30%;
+  /*скрол*/
+  /*overflow-y: scroll;*/
+	max-height: 455px;
+  overflow-y: auto;
+  
+}
+
+.block__element{
+	display:flex;
+	justify-content:flex-end;
+}
+
+.content {
+	border: 5px solid #ece89d;
+	display:flex;
+	flex-wrap: wrap;
+	/*height: 415px;*/
+
+}
+
+.block__element img {
+	justify-content: center;
+	padding: 14px;
+	flex: 0 1 50%;
+	height: 50px;
+	width: 60px;
+}
+
+.block__content .text__icon{
+	display: flex;
+	align-items: center;
+  justify-content: center;
+  font-family: Verdana, sans-serif;
+}
+
+.menu{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+  justify-content: right;
+  top: 30%;
+}
+
+.button{
+	padding: 20px;
+	text-decoration: none;
+	font-family: Verdana, sans-serif;
+	align-items: center;
+  justify-content: center; 
+}
+
+div.block__content{
+	display: block;
+	border: 1px solid transparent;
+	align-items: flex-start;
+}
+
+div.block__content:hover{
+	border-color: grey;
+	display:block;
+	text-align: center;
+}
+
+.element__btn{
+	display:flex;
+	flex-wrap:wrap;
+  min-width:0;
+	text-align: center;
+	font-family: Verdana, sans-serif;
+	cursor: pointer;
+}
+
+div.block__element_13, div.block__element_14, div.block__element_15,
+div.block__element_16, div.block__element_17, div.block__element_18,
+ .text, input[type="checkbox"] {
+  display: none;
```

Comment:
Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide the checkbox in the right way.